cmd/loadTests.go

```diff
@@ -358,27 +289,46 @@ func setup(cmd *cobra.Command, args []string) {
 		metricsInstance.PrintResults()
 	}
 
-	err = createLogDataJSON(
-		"load-tests.json",
-		loadTestsTimestamp,
-		overallCount,
-		threadCount,
-		numberOfUsers,
-		userBatches,
-		"Completed",
-		averageTimeToSpinUpUsers,
-		averageTimeToCreateResources,
-		averageTimeToRunPipelines,
-		userCreationFailureCount,
-		userCreationFailurePercentage,
-		resourceCreationFailureCount,
-		resourceCreationFailurePercentage,
-		pipelineRunFailureCount,
-		PipelineRunFailurePercentage,
-	)
+	/*
+		fetch the below fields:
+		machineName string - the machine on-which the loadTests are run,
+		binaryDetails string - binary details of the program that runs the tests
+	*/
+	machineName, err := os.Hostname()
+	if err != nil {
+		fmt.Printf("error getting hostname: %v\n", err)
```

Comment:
@naftalysh Please use klog.Errorf to print errors instead as in the rest of the code. Thx
```suggestion
		klog.Errorf("error getting hostname: %v\n", err)
```